spec/features/user_sign_in_spec.rb

```diff
@@ -0,0 +1,21 @@
+feature 'User signs in' do
+
+  let(:user) do
+    User.create(username: 'Teeohbee',
+                password: '12345678',
+                email: 'toby@example.com',
+                password_confirmation: '12345678')
+  end
+
+  scenario 'with correct credentials' do
+    sign_in(email: user.email, password: user.password)
+    expect(page).to have_content "Welcome, #{user.username}"
+  end
+
+  def sign_in(email:, password:)
```

Comment:
Unused method argument - `email`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `password`. You can also write as `sign_in(*)` if you want the method to accept any arguments but don't care about them.